### DISCUSSION TITLE
list of manufacturers not available at https://network.igs.org/api/public/

### DISCUSSION THREAD

**Discussion by cknoefel** [International-GNSS-Service/SLM#116] [Q&A]
At the moment there is no database table that supplies the manufacturers:
[https://network.igs.org/api/public/]()

Is this a temporary situation or is the manufacturer list no longer being maintained?

The list is still specified as a possible import option in the documentation:
[https://igs-slm.readthedocs.io/en/latest/commands.html#module-slm.management.commands.import_equipment]()

Thanks for taking care of this.

**Comment by bckohan** [comment]
We haven't updated that production software yet, we will soon in the next few days and the igs manufacturers list will be available on network.igs.org
